src/components/Alert/AttendanceCheckModal/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { useState } from 'react';
+import { Button } from '@/components/ui/Button';
+
+interface Props {
+  highlightText?: string;
+  title: string;
+  button1: string;
+  button2: string;
+  description?: string;
+  button1Click: () => void;
+  button2Click: (
+    selectedPeriod: number | null,
+    reasons: Record<number, string>,
+  ) => void;
+}
+
+const AttendanceCheckModal = ({
+  highlightText,
+  title,
+  button1,
+  button2,
+  description,
+  button1Click,
+  button2Click,
+}: Props) => {
+  const buttonNumbers = [8, 9, 10, 11];
+
+  const [selectedPeriod, setSelectedPeriod] = useState<number | null>(null);
+  const [reasons, setReasons] = useState<Record<number, string>>({});
+
+  const handleReasonChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (selectedPeriod !== null) {
+      setReasons({ ...reasons, [selectedPeriod]: e.target.value });
+    }
+  };
+
+  return (
+    <div className="w-[286px] rounded-[14px] bg-gray-900 pt-4 text-center">
+      <h2 className="font-pretendard mb-1 text-[16px] font-semibold leading-[24px] text-white">
```

Comment:
```suggestion
      <h2 className="!text-h5 text-white">
```
전체적으로 tailwind.config.js를 참고하면서 개발하시면 좋을 거 같아요!

Reply:
2b4f9ba89306341024bf7c91a72f07f5162c2ab7 변경했습니다!